lib/AppInfo/Application.php

```diff
@@ -65,6 +67,10 @@ public function register(IRegistrationContext $context): void {
 		$context->registerEventListener(\OCP\User\Events\UserDeletedEvent::class, UserDeletedListener::class);
 	}
 
+	private function registerPreviewProviders(IRegistrationContext $context): void {
+		$context->registerPreviewProvider(\OCA\Epubviewer\Preview\EPubPreview::class, '/^application\/epub\+zip$/');
```

Comment:
@coderabbitai You're wrong: the second argument of `registerPreviewProvider()` is mandatory.

```php
	/**
	 * Register a preview provider
	 *
	 * It is allowed to register more than one provider per app.
	 *
	 * @param string $previewProviderClass
	 * @param string $mimeTypeRegex
	 * @psalm-param class-string<IProviderV2> $previewProviderClass
	 * @since 23.0.0
	 */
	public function registerPreviewProvider(string $previewProviderClass, string $mimeTypeRegex): void;
```